media-libs/libdmtx/libdmtx-0.7.4.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Library to read and write Data Matrix barcodes"
+DESCRIPTION="libdmtx is a software library that enables programs to read and \
+write Data Matrix (or DataMatrix) 2D barcodes. It runs on several platforms, \
+and can be accessed by multiple languages."
+HOMEPAGE="http://libmtx.sourceforge.net/"
+COPYRIGHT="2005-2011 Mike Laughton"
+LICENSE="libdmtx v1.0"
+REVISION="1"
+SOURCE_URI="https://download.sourceforge.net/libdmtx/libdmtx-$portVersion.tar.gz"
+CHECKSUM_SHA256="686bcfb44e50b96345aa53f41e285f2118a2cf87a333b69b74165a2e7ddca492"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	libdmtx$secondaryArchSuffix = $portVersion
+	lib:libdmtx$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libdmtx${secondaryArchSuffix}_devel = $portVersion
+	devel:libdmtx$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libdmtx$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	rm $libDir/libdmtx.la
+
+	prepareInstalledDevelLib libdmtx
+	fixPkgconfig
+	packageEntries devel $developDir
```

Comment:
Please replace
```
	packageEntries devel $developDir
```
by
```

	if [ -z "$secondaryArchSuffix" ]; then
		maybe_manDir_man3=$manDir/man3
	else
		maybe_manDir_man3=
		rm -rf $manDir/man3
	fi

	packageEntries devel \
		$developDir \
		$maybe_manDir_man3

	rmdir $manDir
```

This will:
* move libdmtx.3 to the devel package;
* discard the man page from any secondary arch builds.

BTW, the isolated "rmdir $manDir" at the end is to make sure we don't remove man pages outside of man3 (without noticing it) should upstream add any in the future.